samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Controllers/AccountController.cs

```diff
@@ -68,6 +68,7 @@ public async Task<ActionResult> LoginAsync(LoginViewModel model)
             {
                 var authnResponse = await _idxClient.AuthenticateAsync(authnOptions).ConfigureAwait(false);
                 Session["idxContext"] = authnResponse.IdxContext;
+                Session[authnResponse?.IdxContext.State] = authnResponse?.IdxContext; // save context in session keyed by state for retrieval by magiclink, see MagicLinkController.Callback.
```

Comment:
This line needs to read

`Session[authnResponse?.IdxContext?.State] = authnResponse?.IdxContext;`

ie Add ? to authnResponse?.IdxContext

Reply:
that said if authnResponse is null we've got problems as well... 🤷 doesn't hurt to add the null check.